stream-chat-android-ui-components/src/main/kotlin/io/getstream/chat/android/ui/message/input/mention/DefaultUserLookupHandler.kt

```diff
@@ -0,0 +1,85 @@
+package io.getstream.chat.android.ui.message.input.mention
+
+import io.getstream.chat.android.client.models.User
+import io.getstream.chat.android.ui.message.input.MessageInputView
+import io.getstream.chat.android.ui.message.input.transliteration.DefaultStreamTransliterator
+import io.getstream.chat.android.ui.message.input.transliteration.StreamTransliterator
+import java.text.Normalizer
+import kotlin.math.min
+
+private const val MAX_DISTANCE = 3
+private val regexUnaccent = "\\p{InCombiningDiacriticalMarks}+".toRegex()
+
+/*
+* Default implementation for MessageInputView.UserLookupHandler. This class ignores diacritics and upper case.
+* Tt uses levenshtein approximation so typos are included in the search. It is possible to choose a transliteration
+* in the class to conversions between languages are possible. It uses https://unicode-org.github.io/icu/userguide/icu4j/
+* for transliteration
+*/
+public class DefaultUserLookupHandler(
```

Comment:
This class is public and moving it to another package creates a breaking change. Can we keep it in the old one?

Reply:
I moved back the old class, but kept the logic in this file to avoid making MessageInputView bigger than already is.